app/export.go

```diff
@@ -40,12 +40,15 @@ func (app *ChainApp) ExportAppStateAndValidators(
 	}
 
 	validators, err := staking.WriteValidators(ctx, app.StakingKeeper)
```

Comment:
[err](https://github.com/cosmos/cosmos-sdk/blob/v0.46.7/x/staking/genesis.go#L15) here is actually nil

Reply:
thank @yihuang for your update